assets/src/onboarding-wizard/index.js

```diff
@@ -90,13 +99,23 @@ Providers.propTypes = {
 domReady( () => {
 	const root = document.getElementById( APP_ROOT_ID );
 
-	if ( root ) {
-		render(
-
-			<Providers>
-				<SetupWizard closeLink={ CLOSE_LINK } finishLink={ FINISH_LINK } appRoot={ root } />
-			</Providers>,
-			root,
-		);
+	if ( ! root ) {
+		return;
 	}
+
+	errorHandler = ( event ) => {
+		// Handle only own errors.
+		if ( /amp-onboarding-wizard(\.min)?\.js/.test( event?.filename ) ) {
```

Comment:
I think this conditional makes more sense like so:

```suggestion
		if ( event.filename && /amp-onboarding-wizard(\.min)?\.js/.test( event.filename ) ) {
```

In other words, it doesn't make sense to possibly pass `null` to the `test()` method.